src/components/incident/AddTicketUrlSignOffAction.tsx

```diff
@@ -23,6 +23,10 @@ const AddTicketUrl: React.FC<AddTicketUrlPropsType> = ({
       dialogSubmitText="Submit"
       dialogCancelText="Cancel"
       dialogButtonText="Add ticket"
+      dialogInputLabel="Valid ticket URL"
+      isDialogInputRequired={false}
+      dialogInputType="url"
+      dialogHelperText="By leaving this field empty, you remove any existing tickets from the selected incidents"
```

Comment:
Clunky text but I can't think of anything better right now. The tickets still exist, it's just the link that is removed.

"Leave this field empty in order to remove ticket urls from the selected incidents" maybe? Hm.

Reply:
I fix!